R/01-estimateMap.R

```diff
@@ -1054,7 +1054,7 @@ modelLocalAvg <- function(data, K, iter, burnin, independent, smoothConst,
   M <- qr(P)$rank
   nknots <- dim(P)[1]
 
-  sV <- smoothCon(s(Latitude, Longitude, m = c(1,0.5),
+  sV <- smoothCon(s(Latitude, Longitude, m = penalty,
```

Comment:
@mgross Currently there is an issue when using Bayesian modelling, see the bug report: https://github.com/Pandora-IsoMemo/DSSM/issues/251#issue-2544957707

One can solve the issue when using here the same penalty for `sV` calculation as for `s` calculation. The mentioned error does not occur anymore. Does this fix make sense or would you suggest another solution?

Reply:
seems fine